eclass/texlive-common.eclass

```diff
@@ -55,9 +55,9 @@ texlive-common_handle_config_files() {
 
 texlive-common_is_file_present_in_texmf() {
 	local mark="${T}/$1.found"
-	[ -d texmf ] && find texmf -name $1 -exec touch "${mark}" \;
-	[ -d texmf-dist ] && find texmf-dist -name $1 -exec touch "${mark}" \;
-	[ -f "${mark}" ]
+	[[ -d texmf ]] && find texmf -name $1 -exec touch "${mark}" \;
```

Comment:
1. `${1}`
2. `find` generally requires a `|| die`, hence you can't use the short-form conditional here